static/app/components/profiling/flamegraph/interactions/useScroll.tsx

```diff
@@ -0,0 +1,47 @@
+import {useCallback} from 'react';
+import {mat3, vec2} from 'gl-matrix';
+
+import {CanvasPoolManager} from 'sentry/utils/profiling/canvasScheduler';
+import {CanvasView} from 'sentry/utils/profiling/canvasView';
+import {FlamegraphCanvas} from 'sentry/utils/profiling/flamegraphCanvas';
+
+export function useCanvasScroll(
```

Comment:
nit: lets sync the file name and this hook

Reply:
Hehe, I just noticed that and doing that in the next PR :D